src/PackingList/PackingList.js

```diff
@@ -48,16 +48,20 @@ const PackingList = ({ packingList, deleteItem, userInfo }) => {
       return (
         <>
           <header className='packing-list-header'>
-            <h1>{tripDetails.title} 
+            <h2 className='packing-list-details'>{tripDetails.title} 
               <BiPencil 
                 className='edit-title' 
                 size={25}
                 onClick={() => setDetailsModalIsOpen(true)}
               />
-            </h1>
-            <h2>{tripDetails.destination}</h2>
-            <h3>{tripDetails.duration} {tripDetails.duration > 1 ? 'days' : 'day'}</h3>
+            </h2>
+            <p className='packing-list-details'>{tripDetails.destination}  |  {tripDetails.duration} {tripDetails.duration > 1 ? 'days' : 'day'}</p>
```

Comment:
I think this looks a little strange using an example of "Wyoming | 2 days". Can we interpolate that a little different, like "2 days in Wyoming" or "Wyoming for 2 days"?

Reply:
Fixed!

Reply:
I like this!